spec/requests/safe_mode_controller_spec.rb

```diff
@@ -10,8 +10,18 @@
       theme.save!
       theme.set_default!
 
+      Fabricate(:admin) # Avoid wizard page
+
+      get '/'
+
+      expect(response.status).to eq(200)
+      expect(response.body).to include("data-theme-id=\"#{theme.id}\"")
+
       get '/safe-mode'
+
+      expect(response.status).to eq(200)
       expect(response.body).not_to include("My Custom Header")
+      expect(response.body).not_to include("data-theme-id=\"#{theme.id}\"")
```

Comment:
@OsamaSayegh I've added a spec to ensure that themes can be disabled in safe mode.